woocommerce/woocommerce-ios
# My Store Analytics: Custom Range tab creation UI flow

Closes: #11971 

Please do not merge until #11975 is merged first. 



## Description


This adds the following flow:
1. Tapping the "Custom Range" button will open the date range picker,
2. Tapping "Add" in the date range picker will create a new tab called "Custom Range" at the end of the analytics tab (with some dummy content). This new tab should be in selected state.
3. Doing the above will also remove the "Custom Range" button.

Note that this is purely UI flow PR. It does not include:

1. The actual content for the "Custom Range" tab.
2. Persistence (e.g: if the app is restarted, "Custom Range" tab will disappear).

## Testing instructions

1. Run app,
3. Tap the Custom Range button on the analytics tab area (the "calendar +" icon)
4. Ensure date range picker is shown,
5. Play around to make sure date selection functionality is working, then tap "Add",
6. Ensure that you're returned to the analytics area, and a new tab called "Custom Range" is shown.
7. Ensure that the Custom Range button is no longer shown.

## Video


https://github.com/woocommerce/woocommerce-ios/assets/266376/f3bb9a96-6171-40f8-a639-cf54f72b4d26
